app/assets/javascripts/global-bar-init.js

```diff
@@ -26,7 +26,8 @@ var globalBarInit = {
 
   urlBlockList: function() {
     var paths = [
-      "^/coronavirus$"
+      "^/coronavirus$",
```

Comment:
Is this line needed? The banner should automatically hide on the page it links to.

Reply:
I'm not sure!

Reply:
I might be brave and try removing it

Reply:
💪

Reply:
Updated